plugins/database/quota/src/org/apache/cloudstack/api/response/QuotaResponseBuilderImpl.java

```diff
@@ -523,14 +523,7 @@ public Date startOfNextDay(Date dt) {
 
     @Override
     public Date startOfNextDay() {
-        Calendar c = Calendar.getInstance();
-        c.setTime(new Date());
-        c.add(Calendar.DATE, 1);
-        c.set(Calendar.HOUR, 0);
-        c.set(Calendar.MINUTE, 0);
-        c.set(Calendar.SECOND, 0);
-        c.set(Calendar.MILLISECOND, 0);
-        return c.getTime();
+        return startOfNextDay(new Date());
```

Comment:
I wouldn't fix this in the same PR. Probably a separate PR for this fix would be better. It's not related to this change.

Reply:
@wido As today our Java guys found bug in Quota plugin related to the code above I ask them to fix it and we send a different PR for it.

Reply:
@wido unrelated code is removed from PR.